components/script/dom/pannernode.rs

```diff
@@ -60,6 +60,15 @@ impl PannerNode {
         if count > 2 {
             return Err(Error::NotSupported)
         }
+        if options.maxDistance < 0. {
+            return Err(Error::NotSupported)
```

Comment:
Should this be a `RangeError`? https://webaudio.github.io/web-audio-api/#dom-pannernode-maxdistance